docs: move CreateQuestion declaration in validations.ts (#619)
blitz-js/blitzjs.com: app/pages/docs/tutorial.mdx

}
```

Next we're going to create a separate file to store the validation schema
for our `QuestionForm`. In the `app/questions` folder create a new file
called `validations.ts` and copy `CreateQuestion` declaration from
`./mutations/createQuestion.ts`. Then update the zod schema so that

> the choice data is accepted in the mutation., and export it so

Reviewer: ```suggestion
the choice data is accepted in the mutation, and export it so
```

> `./mutations/createQuestion.ts`. Then update the zod schema so that
> the choice data is accepted in the mutation., and export it so

Reviewer: ```suggestion
`./mutations/createQuestion.ts`. Then update the zod schema to
validate the choices array, and export it so
```

we can use it for our `QuestionForm` schema in the later step.

```diff
// app/questions/validations.ts
+ import {z} from 'zod'

+ export const CreateQuestion = z
+    .object({
+      text: z.string().nonempty({message: "You must enter a question."}),
+     choices: z.array(z.object({text: z.string()})),
+    })
```

<Card type="info">

>   We create a shared `validations.ts` file because we cannot import
>   anything from a query (or mutation) file other than the query itself
>   into the client. You can read more about why in [Query
>   Usage](https://blitzjs.com/docs/query-usage) and [Mutation
>   Usage](https://blitzjs.com/docs/mutation-usage).

Reviewer: Simplified it a bit, but feel free to disagree 😅
```suggestion
  We create a shared `validations.ts` file so that we can import
  the schemas and use them on the client. We're doing it because
  the code in the `queries/` and `mutations` folders runs only on 
  the server.
```

</Card>

> Now open `app/questions/mutations/createQuestion.ts`, remove old
> CreateQuestion declaration and import new CreateQuestion from
> ../validations.ts. We also need to update the `db.question.create()`

Reviewer: Also, maybe we can remove it while copying it to a separate file?
```suggestion
Now open `app/questions/mutations/createQuestion.ts`, remove old
`CreateQuestion` declaration and import new `CreateQuestion` from
`../validations.ts`. We also need to update the `db.question.create()`
```
